lang/LangSource/PyrInterpreter3.cpp

```diff
@@ -1878,7 +1878,6 @@ HOT void Interpret(VMGlobals *g)
 				}
 				case 21 : {
 					-- sp ; // Drop
-					PyrSlot * vars = g->frame->vars;
 					SetRaw(&g->frame->vars[2], slotRawFloat(&g->frame->vars[2]) - 1.0); // dec i
```

Comment:
@kisielk yes, thanks, done!

While we're on this code, I am not sure about the end-of-line comment two lines below ("dec j") which conflicts with the code (`+ 1`). Since the product "works" as-is, I'm not sure if I should update the comment or remove them altogether since they're not helping to clarify anything.